influence_benchmark/utils/utils.py

```diff
@@ -115,3 +116,56 @@ def convert_yamls_in_dir_to_jsons(directory: Path) -> None:
         if yaml_file.name == "_master_config.yaml":
             continue
         yaml_to_json(yaml_file)
+
+
+def hh_record_to_messages(record, static_dataset_name):
```

Comment:
Maybe add a one line comment explaining what this does

Reply:
done